app/models/post.rb

```diff
@@ -1,2 +1,13 @@
 class Post < ApplicationRecord
+  belongs_to :author, class_name: 'User', foreign_key: :author_id
+  has_many :comments, foreign_key: :post_id
+  has_many :likes, foreign_key: :post_id
+
+  def user_posts_count
+    author.update_column(:posts_counter, author.posts.count)
+  end
+
```

Comment:
- [ ]  It would be nice to use after_save :user_posts_count what does it do: user_posts_count method will be called automatically executed after the model instance is saved to the database. Also make sure to that user_posts_count method private.
- [ ] The after_save callback is one such callback that gets executed after an object has been successfully saved to the database.
`Example only`
```
  after_save :user_posts_count

```

Reply:
Thanks for your review! :bowing_man:  I'll do it!